packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md

```diff
@@ -29,11 +46,11 @@ The following code sample shows how to use `showSignIn()` to handle a token flow
 
 ```javascript
 var signIn = new OktaSignIn({
-   // Assumes there is an empty element on the page with an id of ‘osw-container’
-  el: ‘#osw-container’,
+   // Assumes there is an empty element on the page with an id of 'osw-container'
+  el: '#osw-container',
   clientId: '${clientId of your OIDC app integration}',
   redirectUri: '${redirectUri configured in your OIDC app integration}',
-  baseUrl: ‘https://${yourOktaDomain},
+  baseUrl: 'https://${yourOktaDomain},
```

Comment:
this property value is missing a closing quote mark

Reply:
Fixed

Reply:
This still doesn't look fixed. Also, below here there now looks to be a merge conflict in the file (line 38), and backticks inserted in the file (line 40). Is your text editor doing something weird in that regard?